miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/iap/InAppPurchaseProvider.kt

```diff
@@ -0,0 +1,18 @@
+package com.rakuten.tech.mobile.miniapp.iap
+
+/**
+ * Functionalities related to In-App purchase.
+ */
+interface InAppPurchaseProvider {
```

Comment:
`InAppPurchaseProvider` could be renamed By `IAPBridgeDispatcher` as we name all our bridge dispatcher name like this, what do you think?

Reply:
We already have a `InAppPurchaseBridgeDispatcher` in `miniapp` module to keep some additional setup related to bridge connectivity for this feature. We also have a similar approach like `AdBridgeDispatcher`. 

In this case, I think, it's okay to keep `InAppPurchaseProvider` for this interface.